docker/android/tests/Addition.bp

```diff
@@ -0,0 +1,62 @@
+// special file testing `+` and `+=` operators
```

Comment:
We have very comprehensive tests for the Soong grammar since it's not trivial. It may not be enormously complex, but there's enough edge-cases to be careful.